src/test/java/SkeltalTest.java

```diff
@@ -0,0 +1,37 @@
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import skeltal.Parser;
+import org.junit.jupiter.api.Test;
+import skeltal.Storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SkeltalTest {
+
+    private ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+    private PrintStream standardOut = System.out;
+
+    @BeforeEach
+    public void setUp() {
+        System.setOut(new PrintStream(outputStreamCaptor));
+    }
+
+    @Test
+    public void testParser() {
```

Comment:
Your name for tests should also follow a specific style: see [https://se-education.org/guides/conventions/java/intermediate.html#naming](link).